panda/src/framework/windowFramework.h

```diff
@@ -84,6 +84,15 @@ class EXPCL_FRAMEWORK WindowFramework : public TypedWritableReferenceCount {
   NodePath get_mouse();
   NodePath get_button_thrower();
 
+  NodePath get_a2d_top_center();
+  NodePath get_a2d_bottom_center();
+  NodePath get_a2d_left_center();
+  NodePath get_a2d_right_center();
+  NodePath get_a2d_top_left();
+  NodePath get_a2d_top_right();
+  NodePath get_a2d_bottom_left();
+  NodePath get_a2d_bottom_right();
```

Comment:
Let's make these functions INLINE (mark them so both the implementation and the declaration) and move them to the .I file.

Reply:
Nevermind, looked up more info on inline :)